app/models/calendar.rb

```diff
@@ -7,4 +7,5 @@ class Calendar < ApplicationRecord
   validates :start_year, presence: true
   validates  :months, presence: true
   validates :weekdays, presence: true
+  validates :name, presence: true
```

Comment:
I think we need to take this out for now for our seeds to work. Otherwise looks good.